src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -293,32 +300,40 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
                         onClick={this.handleTableIconFeatureSelect}
                     />
                 </TooltipHost>
-                { this.shouldShowPreviousPageButton() &&
+                {this.shouldShowPreviousPageButton() &&
                     <IconButton
                         className="toolbar-btn prev"
                         title="Previous"
-                        iconProps={{iconName: "ChevronLeft"}}
+                        iconProps={{ iconName: "ChevronLeft" }}
                         onClick={this.prevPage}
                     />
                 }
-                { this.shouldShowNextPageButton() &&
+                {this.shouldShowNextPageButton() &&
                     <IconButton
                         className="toolbar-btn next"
                         title="Next"
                         onClick={this.nextPage}
-                        iconProps={{iconName: "ChevronRight"}}
+                        iconProps={{ iconName: "ChevronRight" }}
                     />
                 }
-                { this.shouldShowMultiPageIndicator() &&
+                {this.shouldShowMultiPageIndicator() &&
                     <p className="page-number">
                         Page {this.state.currentPage} of {this.state.numPages}
                     </p>
                 }
-                { this.state.ocrStatus !== OcrStatus.done &&
+                {this.state.ocrStatus !== OcrStatus.done &&
                     <div className="canvas-ocr-loading">
                         <div className="canvas-ocr-loading-spinner">
                             <Label className="p-0" ></Label>
-                            <Spinner size={SpinnerSize.large} label="Running OCR..." ariaLive="assertive" labelPosition="right"/>
+                            <Spinner size={SpinnerSize.large} label="Running OCR..." ariaLive="assertive" labelPosition="right" />
```

Comment:
The only difference is the spinner label value. Factor out redundant/duplicate code